Tests/PubNubContractTest/Steps/Files/PubNubFilesContractTestSteps.swift

```diff
@@ -156,5 +156,75 @@ public class PubNubFilesContractTestSteps: PubNubContractTestCase {
 
       self.wait(for: [sendFileExpect], timeout: 60.0)
     }
+    
+    When("^I send a file with '(.+)' space id and 'this-is-really-long-message-type-to-be-used-with-publish' message type$") { args, _ in
+      let spaceId = args?.first ?? String()
+      let messageType = "this-is-really-long-message-type-to-be-used-with-publish"
+      
+      let sendFileExpect = self.expectation(description: "Send file Response")
+
+      guard let data = "test file data".data(using: .utf8) else {
+        XCTAssert(false, "Unable prepare file data")
+        return
+      }
+      
+      self.client.fileURLSession = URLSession(
+        configuration: .default,
+        delegate: self.client.fileSessionManager,
+        delegateQueue: .main
+      )
```

Comment:
@jguz-pubnub do we need this one to speed up actual upload?

Reply:
I've discovered so far that the default `.pubnub` background session doesn't suppose to work with unit tests, but everything works fine if I run the sample iOS application with `PubNub` changes from this PR and upload a file. This is why I've used the default session configuration in unit tests so far.

Reply:
But file uses `.background` configuration.
What do you mean that it doesn't support?

Reply:
Sorry, I mislead you. You're right, I also meant `.background` but typed the wrong name in a hurry

Reply:
It looks like a background session is a bad idea for Unit Tests. Thank you, @jguz-pubnub for question on developer forum: https://developer.apple.com/forums/thread/725625